lib/IDE/SyntaxModel.cpp

```diff
@@ -425,6 +425,20 @@ std::pair<bool, Expr *> ModelASTWalker::walkToExprPre(Expr *E) {
   if (isVisitedBeforeInIfConfig(E))
     return {false, E};
 
+  // In SequenceExpr, explicit cast expressions appear twice. Skip them at
+  // operator position.
+  if (auto SE = dyn_cast<SequenceExpr>(E)) {
+    for(unsigned i = 0, e = SE->getElements().size(); i != e; ++i) {
+      auto subExpr = SE->getElement(i);
+
+      if (i % 2 != 0 && isa<ExplicitCastExpr>(subExpr))
```

Comment:
If the pointer for the expression appears twice, can we use a pointer set here to exclude duplications? or do we actually create the same expression twice?

Reply:
That's a good idea. 👍